### PR TITLE
Implementing new commands for keeper-client

### PR DESCRIPTION
**Pull request by pufit**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Added a bunch of custom commands (mostly to make ClickHouse debugging easier).

- getstat - returns the node's stat
- findsupernodes - finds nodes with number of children larger than some threshold for the given path
- deletestalebackups - delete ClickHouse nodes used for backups that are now inactive
- findbigfamily - return the top nodes which have the biggest family - the total number of nodes in the whole subtree

### CODE REVIEW

**Comment by robot-ch-test-poll4** [comment]
This is an automated comment for commit 8dabfc351288a2c5cdf78e7e2b7935c4991da2be with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/51117/8dabfc351288a2c5cdf78e7e2b7935c4991da2be/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
</table>

**Comment by alexey-milovidov** [comment]
Ok, but I don't like that these commands have all words together. What about `find_super_nodes`?

**Review by yariks5s — CHANGES_REQUESTED** [review]
Looks good, need to deal with the problems below and add info to docs: [here](https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/operations/utilities/clickhouse-keeper-client.md).
Also will be great to update tests to check how the new commands work: [here](https://github.com/ClickHouse/ClickHouse/tree/master/tests/integration/test_keeper_client).

File: programs/keeper-client/Commands.h [hunk 1]
```diff
@@ -81,9 +87,63 @@ class GetCommand : public IKeeperClientCommand
 
     void execute(const ASTKeeperQuery * query, KeeperClient * client) const override;
 
-    String getHelpMessage() const override { return "get <path> -- Returns the node's value"; }
+    String getHelpMessage() const override { return "{} <path> -- Returns the node's value"; }
+};
+
+class GetStatCommand : public IKeeperClientCommand
+{
+    String getName() const override { return "getstat"; }
```
Comment (yariks5s) [inline programs/keeper-client/Commands.h:95]:
```suggestion
    String getName() const override { return "get_stat"; }
```
We have `find_super_nodes`, `delete_stable_backups`, `find_big_family`, but `getstat`😕. I think it's better to use `get_stat`.

File: programs/keeper-client/KeeperClient.cpp [hunk 2]
```diff
@@ -177,6 +177,10 @@ void KeeperClient::initialize(Poco::Util::Application & /* self */)
         std::make_shared<SetCommand>(),
         std::make_shared<CreateCommand>(),
         std::make_shared<GetCommand>(),
+        std::make_shared<GetStatCommand>(),
+        std::make_shared<FindSupperNodes>(),
```
[the remark this answers is not in the document]
Comment (pufit) [reply programs/keeper-client/KeeperClient.cpp:181]: Yes, `FindSupperNodes` is a typo.

File: tests/integration/test_keeper_client/test.py [hunk 3]
```diff
@@ -23,10 +24,8 @@ def started_cluster():
         cluster.shutdown()
 
 
-def test_base_commands(started_cluster):
```
Comment (yariks5s) [inline tests/integration/test_keeper_client/test.py:26]: Tests are not running. I think we need to keep `started_cluster` as an argument.

**Review by yariks5s — APPROVED** [review]

**Pull request merged**